portal/db.py

```diff
@@ -86,3 +89,32 @@ def init_app(app):
     app.cli.add_command(init_db_command)
     app.cli.add_command(mock_db_command)
 
+
+def mock_csv():
+    data_filepath = os.path.join(os.path.dirname(__file__), os.pardir, "tests", "portal_users.csv")
+
+    with open(data_filepath, "rb") as f:
+        with get_db() as con:
+
+            with con.cursor() as cur:
+                next(f)
+                
+                cur.copy_from(f, 'users', sep=',', columns=('id', 'email', 'password', 'name', 'role', 'major'))
+                cur.execute("""
+                SELECT password, id FROM users
+                """)
+                records = cur.fetchall()
+                
+                for record in records:
+                
+                    cur.execute(
+                        'UPDATE users SET password = %s WHERE id = %s',
+                        (generate_password_hash(record['password']), record['id'])
+                        )
+                
+            
+                
+                
+                    con.commit()
+
+    
```

Comment:
Nitpick, but there's a lot of extra space in here. Try to clean up your code before committing.